cmake/CreatePlugin.cmake

```diff
@@ -40,12 +40,10 @@ macro(CreatePluginMacro PLUGIN PLUGIN_TYPE)
             )
     endforeach()
 
-    if(_cmp)
-        # See the if statement at the top, this assumes that the plugin type is TEST for now
-        message(VERBOSE "Copying test plugin ${PLUGIN}")
-
+    if(_isTestPlugin)
+        message(VERBOSE "Setting output directory for ${PLUGIN} to ${CMAKE_BINARY_DIR}/Test/plugins")
```

Comment:
I made these changes before discovering the merge conflict -- didn't realize you'd already done this to fix the existing tests since I started w/ the SHA referenced in the CMakeList.txt files that pull this during the build, which was an older commit w/o this fix -- so I kept my changes which I felt were a bit more verbose than yours.